skins/oasis/css/core/breakpoints-layout.scss

```diff
@@ -262,11 +262,13 @@
 				top: 20px;
 
 				.subnav-2 {
+					left: 10px;
 					width: auto;
 				}
 
 				.navbackground {
-					 width: 768px;
+					left: 10px;
```

Comment:
Let me explain - `$padding-width-small-plus` is connected to difference between page's width and content's width - this is not.